udp_test.go

```diff
@@ -0,0 +1,76 @@
+// Copyright (C) 2020 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package jaeger
+
+import (
+	"testing"
+	"time"
+
+	"github.com/spacemonkeygo/monkit/v3"
+	"github.com/stretchr/testify/require"
+
+	"storj.io/monkit-jaeger/gen-go/jaeger"
+)
+
+func TestUDPCollector(t *testing.T) {
+	agentAdrr := "127.0.0.1:0"
+	mock, err := StartMockAgent(agentAdrr)
+	require.NoError(t, err)
+	defer func() {
+		err := mock.Close()
+		require.NoError(t, err)
+	}()
+	collector, err := NewUDPCollector(mock.Addr(), 200, "test", nil)
+	require.NoError(t, err)
+
+	span := &jaeger.Span{
+		TraceIdLow:    monkit.NewId(),
+		SpanId:        monkit.NewId(),
+		OperationName: "test-udp-collector",
+		StartTime:     time.Now().UnixNano() / 1000,
+		Duration:      time.Second.Microseconds(),
+	}
+	collector.Collect(span)
+
+	data := mock.GetBatches()
```

Comment:
If you call this here, you should at least check the value. Otherwise just initialize `data` as an empty variable.